### DISCUSSION TITLE
Annotation layer

### DISCUSSION THREAD

**Discussion by yngrdyn** [hrynko/vue-pdf-embed#108] [Q&A] [answered]
How can I use the annotations layer?
There is no example in the documentation

**Comment by hrynko** [comment, marked as answer]
Hi @yngrdyn,

The annotation layer is enabled by default. The only thing you might need is providing an `imageResourcesPath` property (please check [demo/App.vue](https://github.com/hrynko/vue-pdf-embed/blob/master/demo/App.vue)). Please let me know if I can provide you with more details or if you have ideas what to add to the README file.

**Reply by silverbyte** [reply]
Hi @hrynko

I tried the demo app, but all i get is the rendered PDF. I set the imageResourcesPath to point to the node modules but how am I supposed to add a "hand drawn signature" or "text field" the icons don't load up. Its not clear how I add an annotation (specifically text field or hand signature) would appreciate some more insight. 

much appreciated

**Reply by hrynko** [reply]
Hi @silverbyte,

This library is intended for embedding (rendering) PDF document in the Vue apps. It doesn't allow to edit or add annotations.
